### ISSUE TITLE
"init" complains it can't find the config file?

### ISSUE THREAD

**Issue by nedbat** [sixty-north/cosmic-ray#384]
In an empty directory, in a new Python3.6 virtualenv:
```
$ pip install cosmic-ray
Collecting cosmic-ray
  Using cached https://files.pythonhosted.org/packages/a2/e4/cd1cde639529356da8ed4fa936043bead51a48ea7b534611eb52159baad2/cosmic_ray-3.1.2-py3-none-any.whl
Collecting tinydb>=3.2.1 (from cosmic-ray)
  Using cached https://files.pythonhosted.org/packages/2e/3c/6b07a34632cdffe9071272ec64f8a176fb9148e0a804721a9907ab2f895a/tinydb-3.11.0-py2.py3-none-any.whl
Collecting astunparse (from cosmic-ray)
  Using cached https://files.pythonhosted.org/packages/8b/ea/d38686f1718e307d83673d905dcffd142640a31a217e4e76d1de78f21b20/astunparse-1.5.0-py2.py3-none-any.whl
Collecting qprompt (from cosmic-ray)
Collecting decorator (from cosmic-ray)
  Using cached https://files.pythonhosted.org/packages/bc/bb/a24838832ba35baf52f32ab1a49b906b5f82fb7c76b2f6a7e35e140bac30/decorator-4.3.0-py2.py3-none-any.whl
Collecting docopt-subcommands>=2.3.0 (from cosmic-ray)
  Using cached https://files.pythonhosted.org/packages/53/14/2e2398c5684657a4dd0c4bb6c2d7c196de8d63b7096345153f21ad1bb553/docopt_subcommands-2.3.1-py2.py3-none-any.whl
Collecting pathlib (from cosmic-ray)
Collecting spor (from cosmic-ray)
  Using cached https://files.pythonhosted.org/packages/5c/02/8ff022acc9b7c66c733f7795b4277ba64b7e9c2e6912ab41affc97018000/spor-1.0.2-py3-none-any.whl
Collecting pyyaml (from cosmic-ray)
Collecting stevedore (from cosmic-ray)
  Using cached https://files.pythonhosted.org/packages/a1/d9/93a975469c53a9ee85de9ec0deb12345aa777748b4c263860668592344fe/stevedore-1.29.0-py2.py3-none-any.whl
Requirement already satisfied: wheel<1.0,>=0.23.0 in /usr/local/virtualenvs/cosmic/lib/python3.6/site-packages (from astunparse->cosmic-ray) (0.31.1)
Collecting six<2.0,>=1.6.1 (from astunparse->cosmic-ray)
  Using cached https://files.pythonhosted.org/packages/67/4b/141a581104b1f6397bfa78ac9d43d8ad29a7ca43ea90a2d863fe3056e86a/six-1.11.0-py2.py3-none-any.whl
Collecting docopt (from docopt-subcommands>=2.3.0->cosmic-ray)
Collecting pbr!=2.1.0,>=2.0.0 (from stevedore->cosmic-ray)
  Using cached https://files.pythonhosted.org/packages/69/1c/98cba002ed975a91a0294863d9c774cc0ebe38e05bbb65e83314550b1677/pbr-4.2.0-py2.py3-none-any.whl
Installing collected packages: tinydb, six, astunparse, qprompt, decorator, docopt, docopt-subcommands, pathlib, pyyaml, spor, pbr, stevedore, cosmic-ray
Successfully installed astunparse-1.5.0 cosmic-ray-3.1.2 decorator-4.3.0 docopt-0.6.2 docopt-subcommands-2.3.1 pathlib-1.0.1 pbr-4.2.0 pyyaml-3.13 qprompt-0.12.0 six-1.11.0 spor-1.0.2 stevedore-1.29.0 tinydb-3.11.0
$ cosmic-ray init config.yml my_session
Error loading configuration from config.yml
[Errno 2] No such file or directory: 'config.yml'
```

I thought init would create the yaml file?  I used the exact first command from the quickstart. I must have misunderstood something.

**Comment by abingham** [comment]
Hmm. This was supposed to have been fixed for issue #383. Not sure what happened there. See the discussion there for details on what to do.

I'll fix the documentation when I get back from vacation in a few weeks. Sorry for the confusion.

**Comment by abingham** [comment]
The docs should be up to date as of dc3f0616dbffdceae6c25dbfe4f3200429642e63. If readthedocs looks out of date on your browser, you might need to do an un-cached refresh.

I'm going to close this, but please re-open the issue if it doesn't seem fixed to you. Thanks for reporting it!
